spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] Following the project requirements, you should use `type: request` as shown in the example code below OR use `config.infer_spec_type_from_file_location!` in the Rspec config file (in that case just placing a file in the `spec/requests` dir will be enough).
```ruby
RSpec.describe "Users", type: :request do
   ...
 end
```